rpc/core/blocks.go

```diff
@@ -193,6 +193,7 @@ func (env *Environment) BlockResults(_ *rpctypes.Context, heightPtr *int64) (*ct
 		FinalizeBlockEvents:   results.Events,
 		ValidatorUpdates:      results.ValidatorUpdates,
 		ConsensusParamUpdates: results.ConsensusParamUpdates,
+		AppHash:               results.AppHash,
```

Comment:
Is this field returned via RPC? Or at least was it before (e.g., v0.38.x)?

Reply:
What about `LastResultsHash`? Should we include it too?